doc/format.rst

```diff
@@ -175,8 +175,9 @@ respective types.
   y
       Description of parameter `y` (with type not specified).
 
-Enclose variables in single backticks.  The colon must be preceded
-by a space, or omitted if the type is absent.
+The colon must be preceded by a space, or omitted if the type is absent.
+When referring to a parameter in the description field or elsewhere within
```

Comment:
Perhaps this would be simpler as "referring to any parameter of the function being documented", or something along that vein?

Reply:
So basically remove "in the description field or...". That's probably fine. I think I wanted to point out specifically that this advice was not limited to references within the parameter documentation, but maybe it's better overall to just keep it simple.

Reply:
Where else can the variable live? Returns? Class docs, etc.?

Reply:
I'm not sure I understand. Is the question about what constitutes a "parameter", where else a parameter might be mentioned, or something else?